pkgs/top-level/splice.nix

```diff
@@ -102,7 +116,7 @@ let
     , pkgsHostTarget
     , pkgsTargetTarget
     } @ args:
-    if actuallySplice then spliceReal args else pkgsHostTarget;
+    if true then spliceReal args else pkgsHostTarget;
```

Comment:
This isn't intentional, right?

Reply:
It's so ofborg uses the splicing path

Reply:
Makes total sense. I'm very interested in this (and similar) PRs!